packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart

```diff
@@ -138,11 +139,12 @@ abstract class ImagePickerPlatform extends PlatformInterface {
   ///
   /// In Android, the MainActivity can be destroyed for various reasons. If that happens, the result will be lost
   /// in this call. You can then call [retrieveLostData] when your app relaunches to retrieve the lost data.
-  Future<PickedFile> pickImage({
+  Future<PickedImage> pickImage({
```

Comment:
If possible to avoid a breaking change here it's highly preferable, I'd consider either deprecating this method and introducing a new one, or keeping this one as the way to get just the image and introduce and additional one for getting an image and a thumbnail